assets/css/_ladder.scss

```diff
@@ -1,4 +1,6 @@
 .m-ladder {
+  grid-column: span 1;
+  grid-row: 2 / 3;
```

Comment:
The grid lines can be given names instead of the default indices. Idk what best practice is, though.

Reply:
Named rows definitely seem nice.